static/app/views/dashboardsV2/dashboard.tsx

```diff
@@ -620,7 +620,7 @@ const WidgetContainer = styled('div')`
 
 const GridItem = styled('div')`
   .react-resizable-handle {
-    z-index: 1;
+    z-index: 2;
```

Comment:
@edwardgou-sentry It'll be a good idea to store and organize these z indices in a file to keep track of the layers of various `<div/>`s for dashboards. Not something to be done for this pull request.